File: semestre-1/algoritmos/lista-1/python/MR-Alg-01-Ex-04.py
largura = float(input("Largura do terreno (em metros): "))
profundidade = float(input("Profundidade do terreno (em metros): "))

area = largura * profundidade
hectares = area / 10000

print(f"O terreno possui {hectares} hectar{hectares != 1 and 'es' or ''}.")

File: semestre-1/algoritmos/lista-1/c/MR-Alg-01-Ex-04.c
#include <stdio.h>

void main() {
   float largura, profundidade;

    printf("Largura do terreno (em metros): ");
    scanf("%f", &largura);

    printf("Profundidade do terreno (em metros): ");
    scanf("%f", &profundidade);

    float area = largura * profundidade;
    float hectares = area / 10000;
    printf("Hectares do terreno: %f\n", hectares);
}